src/App.vue

```diff
@@ -96,9 +124,19 @@ export default {
       return sessionStorage.getItem('BUSINESS_ID')
     },
 
+    /** The NR Number string. */
+    nrNumber (): string {
+      // change _ to %20 for query purposes
+      return sessionStorage.getItem('NR_NUMBER')?.replace('_', '%20')
```

Comment:
Interesting! Doesn't it apply this automatically in the browser if entered with a space?

Reply:
The Auth API, for one, doesn't respond with the requested data if I enter "NR xxxxxxx" instead of "NR%20xxxxxxx".

In the browser URL, the space is automatically converted to %20. (And also when  you cut and paste to/from the browser URL.)